Assets/Scripts/Game/Entities/DaggerfallEntityBehaviour.cs

```diff
@@ -52,10 +66,19 @@ public DaggerfallEntity Entity
         void Start()
         {
             SetEntityType(EntityType);
+            playerMotor = GameManager.Instance.PlayerMotor;
         }
 
         void Update()
         {
+            // Wait until game has started and the game time has been set.
```

Comment:
Ideally move game started check inside your if (EntityType == EntityTypes.Player) block. Putting the game started check at front of method actually blocks the intended entity set and update, which should happen whether game is running or not.